rust/agama-migrate-wicked/README.md

```diff
@@ -0,0 +1,11 @@
+# Migrate wicked
+This project creates a `migrate-wicked` binary which is able to parse wicked xml configs and send them to a NetworkManager dbus service.
+## Architecture
+`migrate-wicked` uses agama as a library to communicate the parsed network state to NetworkManager
+but the binary is completely independent of any agama services and can be run standalone
+## Obtaining wicked xml config
+A wicked config xml can be generated by running `wicked show-config > wicked.xml` which can then be passed to `migrate-wicked`
+## Testing
+Running the migration on a live system isn't currently recommended. To test it's recommended to use a container
+(instructions how to set one up can be found [here](https://github.com/openSUSE/agama/blob/master/rust/agama-cli/doc/backend-for-testing.md))  
```

Comment:
Just a comment: I think this container is actually quite heavy for our needs. As we simply need a container with NetworkManager installed, or?

Reply:
You're right. I think for now and just for testing, it's not that important but I guess we could write a simpler script that skips all the unnecessary steps. Though for now I'd keep the image the same and maybe later build our own minimal container. WDYT?